selectel/resource_selectel_mks_cluster_v1.go

```diff
@@ -0,0 +1,560 @@
+		Delete:   resourceMKSClusterV1Delete,
+		Importer: nil,
+		CustomizeDiff: customdiff.All(
+			customdiff.ComputedIf(
+				"updated_at",
+				func(d *schema.ResourceDiff, meta interface{}) bool {
+					var hasChange bool
+					if d.HasChange("maintenance_window_start") {
+						hasChange = true
+					}
+					if d.HasChange("enable_autorepair") {
+						hasChange = true
+					}
+					if d.HasChange("enable_patch_version_auto_upgrade") {
+						hasChange = true
+					}
+
+					return hasChange
+				}),
+			customdiff.ComputedIf(
+				"maintenance_window_end",
+				func(d *schema.ResourceDiff, meta interface{}) bool {
+					return d.HasChange("maintenance_window_start")
+				}),
+		),
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"kube_version": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"region": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					ru1Region,
+					ru2Region,
+					ru3Region,
+					ru7Region,
+					ru8Region,
+				}, false),
+			},
+			"nodegroups": {
+				Type:     schema.TypeSet,
+				Required: true,
+				ForceNew: false,
+				Set:      hashNodegroups,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"availability_zone": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"count": {
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+						"keypair_name": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"affinity_policy": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"cpus": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"ram_mb": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"volume_gb": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Computed: true,
+						},
+						"volume_type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+						"local_volume": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+						"flavor_id": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+						"cluster_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"created_at": {
```

Comment:
I guess that `created_at`, `updated_at` fields are useless for a provider user. Same for cluster, nodes `created_at`, `updated_at` fields.